shm/README.md

```diff
@@ -27,7 +27,7 @@ ___
 
 ## Zadanie 4
 
-Napisz klasę `Map`, która będzie posiadała `std::vector<Island&>` przechowujący wszystkie wyspy na mapie, oraz zmienną `Island* currentPosition_` określającą aktualną pozycję gracza na mapie.
```

Comment:
I think it should be pointer or std::unqiue_ptr but I don't remember, need to check :D

Reply:
I don't know what you ment with the assignment. I ment that reference can't be a type in vector because it is not assignable. https://stackoverflow.com/questions/922360/why-cant-i-make-a-vector-of-references

Reply:
I think it's a good suggestion. I only wonder if Map should be an owner of all Islands? I don't know now how it's gonna be used, but it seems ok.